src/components/calcite-link/calcite-link.scss

```diff
@@ -1,19 +1,26 @@
+:host {
+  --calcite-link-blue-underline: #{rgba($h-bb-070, 0.4)};
+}
+
+:host([theme="dark"]) {
+  --calcite-link-blue-underline: #{rgba($d-bb-420, 0.4)};
+}
```

Comment:
@julio8a these vars already exist in global.scss

Reply:
ty